tools/rosbash/rosbash

```diff
@@ -1015,3 +1026,4 @@ complete -F "_roscomplete_rossrv" "rossrv"
 complete -F "_roscomplete_roscreate_pkg" "roscreate-pkg"
 complete -F "_roscomplete_roswtf" -o filenames "roswtf"
 complete -F "_roscomplete_rosconsole" "rosconsole"
+complete -F "_roscomplete_file" "rosmv"
```

Comment:
Please add a newline at the end of the file.

Reply:
Added new line